tests/functional/job-submission/01-job-nn-localhost/db.sqlite3

```diff
@@ -6,7 +6,7 @@ CREATE TABLE inheritance(namespace TEXT, inheritance TEXT, PRIMARY KEY(namespace
 INSERT INTO inheritance VALUES('root','["root"]');
 INSERT INTO inheritance VALUES('foo','["foo", "root"]');
 CREATE TABLE workflow_params(key TEXT, value TEXT, PRIMARY KEY(key));
-INSERT INTO workflow_params VALUES('cylc_version', '8.0.0.dev');
+INSERT INTO workflow_params VALUES('cylc_version', '8.0.0');
```

Comment:
Test was failing because it turns out `8.0.dev` is less than `8.0rc2` or even `8.0a0`!

Reply:
That was actually a mistake on my part - I had meant to change it to 8.0.3.dev, not 8.0.0.dev.  But I guess your solution will work just as well!